apps/foundation-demo/src/app/thank-you/components/print-order-summary/print-order-summary.component.html

```diff
@@ -0,0 +1 @@
+<button type="button" class="button large expanded" (click)="printOrderSummary()">Print Order Summary</button>
```

Comment:
Does `print-order-summary` need to be its own component if it's only used on the thank you page, and the only functionality it has is ```printOrderSummary() {
    console.log('print order summary');
  }```?

Reply:
I think it should be its own component. I could ask a similar question: why is `print-order-summary` on the page at all if all it does is console.log something? Answer: it's a placeholder for print order summary functionality in the future. And when that functionality is implemented, it shouldn't be in the `thank-you` component, because of separate of concerns.